app/Dockerfile

```diff
@@ -7,15 +7,20 @@ ENV MM_VERSION=4.4.1
 # Build argument to set Mattermost edition
 ARG edition=entreprise
 
-# Install some needed packages
-RUN apt-get update \
-    && apt-get -y --no-install-recommends install \
-      curl \
-      jq \
-      netcat \
-      ca-certificates \
-    && rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin
 
+RUN apk add --no-cache \
+	netcat-openbsd \
```

Comment:
Some maniac style comment here : can you order package list by alphabetical order and use the same indent please ? :)